cmd/sovereignnode/config/sovereignConfig.toml

```diff
@@ -39,6 +39,9 @@
     ]
 
 [OutGoingBridge]
+    # This flag enables or disables the outgoing bridge service connection.
+    # When enabled, the node will process and forward outgoing transactions to the bridge service.
```

Comment:
Node will process the outgoing operations nevertheless, only that it will not send if disabled